### ISSUE TITLE
Update to pyjwt has added breaking changes to released version

### ISSUE THREAD

**Issue by fayebutler** [yunojuno/django-request-token#65]
Breaking change added to pyjwt v2.10.0 here: https://github.com/jpadilla/pyjwt/issues/1017

Since this library has not fixed a version of pyjwt the most up to date is installed and this has caused breaking changes in our build.

**Comment by djm** [comment]
@fayebutler Yes, we hit this yesterday too. We don't have immediate capacity to fix it this week, so I would recommend pinning your version of pyjwt in your local setup for now.

We are also open to fixes from the community.

**Comment by djm** [comment]
Closing this issue as resolved.
